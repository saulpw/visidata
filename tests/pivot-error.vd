sheet	col	row	longname	input	keystrokes	comment
			open-file	sample_data/test.jsonl	o	
test	key1		type-date		@	
test	qty		type-int		#	
test	amt		type-float		%	
test	qty		aggregate-col	sum	+	
test	key1		key-col		!	
test	key2		pivot		W	
test_pivot_key2_sum	key1		sort-desc		]	
