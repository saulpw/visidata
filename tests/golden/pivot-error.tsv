key1	foo	baz	bar
2018-10-20	0	0	0
2018-07-27	0	256	0
2017-12-25	0	0	16
2016-01-01	1	0	0
date(None)	0	36	0
#ERR	0	0	0
